### ISSUE TITLE
Clarify download being free or not?

### ISSUE THREAD

**Issue by rawtaz** [imapsync/imapsync#257]
I'm trying to establish whether or not imapsync is really free to download and use, but it's not entirely clear. In this discussion I'm using Windows 10 as the intended platform.

**Looking for installation instructions starting from the GitHub repository**

- If you go to this repository and the [README.md](https://github.com/imapsync/imapsync/blob/master/README.md) it says "Thanks in any case, even no money, I'm ok with that!" which suggests you offer imapsync downloads without requiring a purchase or donation.
- In the [install.d folder] with installation instructions, there's no Windows specific file, so instead I read the [installation instructions for any platform](https://github.com/imapsync/imapsync/blob/master/INSTALL.d/INSTALL.ANY.txt).
- This file does have a heading "Installing imapsync on WINDOWS", which says "Read the file README_Windows.txt" (which there isn't one in that same folder and "Also available at [https://imapsync.lamiral.info/README_Windows.txt](https://imapsync.lamiral.info/README_Windows.txt)".
- Reading that referenced README_Windows.txt file on the main website, it says (under heading "A.1) Get imapsync") to "Get imapsync at https://imapsync.lamiral.info/. You'll then have access to a zip archive file named imapsync_1.xxx.zip where 1.xxx is the imapsync release number."
- So we go to https://imapsync.lamiral.info and search for a download link, but the only relevant occurrences of the word "download" on that page is the part under the "Buy imapsync and support" heading, which says "Instantly at the end of the payment, Paypal will proposed you to go back to the site, via a link to gilles@lamiral.info (it's my paypal account name), this link brings you to imapsync download". Besides this, all other occurences of "download" just refers the user to the installation instructions, which just leads us back to https://imapsync.lamiral.info/README_Windows.txt .

**Looking for installation instructions starting from the main website**

- Similar to the above, searching for "download" or installation instructions on the main website in the end just points you to the https://imapsync.lamiral.info/README_Windows.txt file, which tells you to go to the main website to get imapsync, which takes us back to square one.

**One additional note:** I did download the latest release from the GitHub repository, in this case https://github.com/imapsync/imapsync/archive/1.977.zip , but there's no imapsync.exe in there from what I can tell.

**So, in summary:** Where can one download imapsync for Windows? Am I missing something in terms of instructions or download links?

All this said, I'm not suggesting everyone should pry on your work - I'm just trying to establish what the status is regarding the above. Thanks!

**Comment by gilleslamiral** [comment]
The imapsync licence is  "No limits to do anything with this work and this license."  
https://imapsync.lamiral.info/LICENSE

Why you don't find a direct download link on the upstream site https://imapsync.lamiral.info/ is because:

1.  I earn my leaving from imapsync buyers but less likely on donators, as a measured fact.
2. Donations work 1/100 less than payments, ie, for $1 from donators I get $100 of buyers.
3. I don't make the github release, Nicolas does, and Nicolas doesn't include the .exe binary in the repository, it is a common rule in free and open source software guidelines. I do not blame Nicolas about this.
4. Nicolas does a good job, a basic copy of imapsync mainstream repository where releasecheck is off and whatever he wants, like exe trash, I'm ok with that.

To answer your questions:
- Windows binary exe is here in the zip file: https://imapsync.lamiral.info/dist/
- Don't worry, my income won't suffer, as people don't read. The no-dowload link on the main imapsync web site is just a payment incentive. It works. It works enough to make me concentrate my work life on imapsync and their users.

Feel free to share you thoughts about this, free software and business is a good topic I like to talk about.

**Comment by rawtaz** [comment]
Thank you for an elaborate and well structured reply :)

I have nothing against any of what you write, although I don't really see the point of not distributing the .exe on GitHub if it's available on the main site.

Thanks for the link to the download. Just to clarify; Is your intention that when people look for download links on the main site, then if they don't want to buy or donate, they're expected to mail or contact you some other way and ask for the download link, which you then give them?

**Comment by gilleslamiral** [comment]
> I have nothing against any of what you write, although I don't really see the point of not distributing the .exe on GitHub if it's available on the main site.
> 

That's a classical open-source directive of safety, you never know what there is in a binary.

> Just to clarify; Is your intention that when people look for download links on the main site, then if they don't want to buy or donate, they're expected to mail or contact you some other way and ask for the download link, which you then give them?

Yes. 
When people don't want to pay they don't, no matter the licence, no matter the protections, no matter the who knows, they find a way to have the software tool they need. My way to this issue is just "do what you want as usual or just ask me and I'll give you my stuff for just contacting me. And I'm ok and happy with that."

**Comment by Friedman10** [comment]
Thank you for your work.

**Comment by akokarev** [comment]
You hide the distribution well, I was tired of searching, so I asked GPT and found the /dist2/ folder. I was curious about where GPT found this link. That's when I discovered this discussion. Well, I have to give credit to the author. However, aren't you worried that users might find this utility on a third-party website along with a virus, resulting in a negative experience?

**Comment by gilleslamiral** [comment]
Currently, any search with "imapsync download" gives imapsync GitHub repository as 2nd link and dist2/ as third link.  It varies, sometimes dist2/ is the first link given in response. 

> However, aren't you worried that users might find this utility on a third-party website along with a virus, resulting in a negative experience?

I have no worries, at all. 
It never happened in 24 years :-)
